src/AdminDetail/index.js

```diff
@@ -1,3 +1,3 @@
-// Login Details of the user
+// Login Constants of the user
```

Comment:
@Vishalguptag - Please comment in a bit of detail. For example: Please use below username and password for login.

Reply:
okkk sir